serverless.js

```diff
@@ -56,6 +41,11 @@ module.exports = {
     memorySize: 512,
     region: AWS_REGION,
     stage: SLS_STAGE,
+    httpApi: {
```

Comment:
When creating `AWS::ApiGatewayV2::Api` cors configuration is at the provider level.